### PR TITLE
feature add /  probabilidades

### PR DESCRIPTION
**Pull request by SebaSatelier**
add helper probabilidades.

maybe we could will change the name of helpers to english?

### CODE REVIEW

File: src/helpers/probabilidades
Comment (manduinca) [inline src/helpers/probabilidades:1]: don't forget the extension file.

**Review by manduinca — CHANGES_REQUESTED** [review]
Please fix it.

**Comment by SebaSatelier** [comment]
add extension file,
change helper name to probabilities

**Review by manduinca — APPROVED** [review]
I think these comments are not necessaries.

**Review by FacundoPucheta — APPROVED** [review]

**Pull request merged**